classes/booking_manager.php

```diff
@@ -390,4 +393,12 @@ public function process() {
 
         return true;
     }
+
+    /**
+     * Sets if the confirmation emails should be suppressed
+     * @param bool $val
+     */
+    public function set_suppress_email($val) {
+        $this->suppressemail = $val;
```

Comment:
It might not hurt to add some additional hints, and clarify what the value would be without the phpdocs.
```suggestion
    public function set_suppress_email(bool $suppressed) {
        $this->suppressemail = $suppressed;
```